app/views/content/is-teaching-right-for-me/maths/maths/_article.html.erb

```diff
@@ -0,0 +1,95 @@
+      <p>You'll teach the national curriculum. The themes you're likely to cover include: </p>
+      <p>Themes you’ll cover for 11 to 16 year olds (key stage 3 and 4):</p>
+      <ul class="vertical-tags">
+        <li class="tag">Numbers</li>
+        <li class="tag">Algebra</li>
+        <li class="tag">Ratio, proportion and rates of change</li>
+        <li class="tag">Geometry and measures</li>
+        <li class="tag">Probability and statistics</li>
+      </ul>
+
+      <p>Themes you’ll cover for 16 to 18 year olds (key stage 5) - A or AS level core maths:</p>
+      <ul class="vertical-tags">
+        <li class="tag">Analysis of data</li>
+        <li class="tag">Maths for personal finance</li>
+        <li class="tag">Estimation</li>
+        <li class="tag">Critical analysis of given data and models</li>
+      </ul>
+    <p><a href="https://www.gov.uk/government/publications/national-curriculum-in-england-mathematics-programmes-of-study/national-curriculum-in-england-mathematics-programmes-of-study#key-stage-3">Read the full national curriculum for more information about teaching maths in secondary schools</a>.</p>
+    <p>You may also get the opportunity to teach other subjects, particularly for 11 to 16 year olds and in some cases for post-16. This will vary depending on individual schools.
+    </div>
+</section>
+<section class="col col-845 inset">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/images/content/subjects/rotna-roy.jpeg",
+          heading: "Empowering my pupils",
+          text: "I enjoy showing my pupils that maths is essential in everyday life. Whether it is identifying patterns, interpreting data, managing finances, or understanding the spaces and structures around us, everything uses maths.",
+          accreditation: "Rotna Roy, Maths Lead at a school in London",
+        }
+      ],
+      colors: %w[yellow],
+      numbered: false
+    ) %>
+  </section>
+
+<section class="col col-space-m col-720">
+<h2 class="heading-s">Funding for training to teach maths</h2>
+<p>To train to be a maths teacher, you will need to do a teacher training course. </p>
+<p>Tax-free bursaries of £28,000 and scholarships of £30,000 are available for <a href="/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=maths&button=#funding-widget">eligible trainee maths teachers</a>. </p>
+<p>You can find out more about <a href="https://teachingmathsscholars.org/home">the maths scholarship from The Institute of Mathematics and its Applications, and partners</a>.</p>
+<p>You can receive this alongside a tuition fee and maintenance loan. </p>
+<p>You may also be able to get extra funding support if you are: </p>
+<ul>
+<li>a parent or carer </li>
+<li>disabled </li>
+</ul>
+<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.</p>
+<h2 class="heading-s">Check your qualifications</h2>
```

Comment:
This h2 feels a bit different in that it starts with a verb - worth making h2s consistent?